pipelines/_ingredients/MetricsMiscCalibrateImage.yaml

```diff
@@ -0,0 +1,13 @@
+# TODO DM-40388: This file will be merged into MetricsMisc.yaml, once the new pipeline is vetted.
+# Miscellaneous metrics for Alert Production
+# In the future, these might be placed in task-specific pipelines (for debugging)
+# or grouped by their datasets (to optimize expensive Butler reads)
+imports:
+  - location: $AP_VERIFY_DIR/pipelines/_ingredients/MiscMetrics.yaml
```

Comment:
Same question about the dependency direction.

Reply:
Same answer as above.